src/components/standalone/monitoring/NetifyInformaticsContent.vue

```diff
@@ -0,0 +1,200 @@
+<!--
+  Copyright (C) 2024 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+import { useI18n } from 'vue-i18n'
+import FormLayout from '@/components/standalone/FormLayout.vue'
+import { useNotificationsStore } from '@/stores/notifications'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { faArrowUpRightFromSquare, faCopy } from '@fortawesome/free-solid-svg-icons'
+import {
+  NeSkeleton,
+  NeTextInput,
+  getAxiosErrorMessage,
+  NeInlineNotification,
+  NeTooltip,
+  NeButton,
+  NeLink
+} from '@nethesis/vue-components'
+import { NeToggle } from '@nethesis/vue-components'
+import { ubusCall } from '@/lib/standalone/ubus'
+import { AxiosError } from 'axios'
+
+const notificationsStore = useNotificationsStore()
+const { t } = useI18n()
+
+interface Form {
+  status: boolean
+  uuid: string
+}
+
+const form = ref<Form>({
+  status: false,
+  uuid: ''
+})
+
+const objError = {
+  notificationTitle: '',
+  notificationDescription: '',
+  uuid: ''
+}
+
+const error = ref({ ...objError })
+const errorLoadingConfiguration = ref({ ...objError })
+const errorSaving = ref({ ...objError })
```

Comment:
This part is really hard to understand

`error` is effectively not used, if not for line 162, however the value is never set, so what is this for?
`errorLoadingConfiguration` and `errorSaving` are basically the same, there's no difference between them, they can be the same value